queries/go/locals.scm

```diff
@@ -0,0 +1,62 @@
+(
+    ;(comment)* @doc
+    (function_declaration
+        name: (identifier) @definition.function) ;@function 
+    ;(#strip! @doc "^//\\s*")
+    ;(#set-adjacent! @doc @function)
+)
+
+(
+    ;(comment)* @doc
+    (method_declaration
+        name: (field_identifier) @definition.method); @method
+    ;(#strip! @doc "^//\\s*")
+    ;(#set-adjacent! @doc @method)
```

Comment:
We might want to implement these too.
What `#set-adjacent!` is meant to do ?

Reply:
I have no idea. I guess since it's a tag query this should format the final content of the tag. Or system works differently so we could probably just omit it.